src/js/BrowserUtils.js

```diff
@@ -15,6 +15,7 @@
 goog.provide('axs.browserUtils');
 
 /**
+ * Use Webkit matcher when matches() is not supported.
```

Comment:
Ah, I see you were just adding missing documentation, this makes sense to me now.